### ISSUE TITLE
Run or Debug dropdown should be triggered via up / down arrows

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#129520]
Testing #129384

Focus the Run or Debug dropdown chevron. Press `up` or `down` -> the dropdown is not opened.
Notice how the dropdown gets nicely opened when the focus is on the Start Debugging action in the Debug view title area.

It seems like the Terminal uses the same widget and I can reproduce this same issue in the terminal fyi @Tyriar

**Comment by Tyriar** [comment]
Thinking some more about this I'm not sure it is expected that up/down does anything here since it's not a regular `select` box but a button that opens a menu. It uses `DropdownMenuActionViewItem` which is used by the overflow menu as well so that and any other usage would inherit that behavior if we did it.

It's unclear who the right owner is for this.

**Comment by weinand** [comment]
@Tyrar no, the Run/Debug item uses an `DropdownWithDefaultActionViewItem` because its logic is quite different than the   `DropdownMenuActionViewItem`. So adding the up/down functionality here would nit affect other places.

**Comment by roblourens** [comment]
This works now
